src/decorators/bind.ts

```diff
@@ -98,10 +122,24 @@ export function bind(...args: any[]): any {
   });
 }
 
+function getConverter(isShorthand: boolean, args: any[]): BindSuperConfig<any>['convert'] {
+  if (isShorthand) {
+    return args.length === 2 ? {binding: args[1]} : null;
+  }
+  if (args[0].convert instanceof Function || typeof args[0].convert === 'string') {
+    return {property: args[0].convert};
+  }
+  return args[0].convert || null;
```

Comment:
Prefer `??` over `||`?

Reply:
I would like the result to be null for any falsy value. (But those should not be possible to begin with if a typescript compiler checked the code.)